MSCL/source/mscl/MicroStrain/MIP/MipNode_Impl.h

```diff
@@ -307,6 +307,19 @@ namespace mscl
         //    - <Error_Connection>: A connection error has occurred with the InertialNode.
         virtual GnssReceivers getGnssReceiverInfo() const;
 
+        //Function: getSupportedSensorRanges
+        //    Gets the supported sensor ranges for this device.
+        //    Sends the Supported Ranges command (0x0C,0x53) to the device. 
```

Comment:
Not sure if it's line endings or just an extra space, but there's a space after the period

Reply:
It was a space, good catch!